Doc/reference/compound_stmts.rst

```diff
@@ -151,20 +151,19 @@ The :keyword:`!for` statement
    single: : (colon); compound statement
 
 The :keyword:`for` statement is used to iterate over the elements of a sequence
-(such as a string, tuple or list) or other iterable object:
+(such as a string, tuple, or list) or other iterable object:
 
 .. productionlist:: python-grammar
-   for_stmt: "for" `target_list` "in" `starred_list` ":" `suite`
+   for_stmt: "for" `target_list` "in" `starred_expression_list` ":" `suite`
            : ["else" ":" `suite`]
 
-The ``starred_list`` expression is evaluated once; it should yield an
-:term:`iterable` object.  An :term:`iterator` is created for that iterable.
-The first item provided
-by the iterator is then assigned to the target list using the standard
-rules for assignments (see :ref:`assignment`), and the suite is executed.  This
-repeats for each item provided by the iterator.  When the iterator is exhausted,
-the suite in the :keyword:`!else` clause,
-if present, is executed, and the loop terminates.
+The expression following ``in`` is evaluated once; it should yield an
+:term:`iterable` object. An :term:`iterator` is created for that iterable. The
+first item provided by the iterator is then assigned to the target list using
+the standard rules for assignments (see :ref:`assignment`), and the suite is
+executed. This repeats for each item provided by the iterator. When the iterator
+is exhausted, the suite in the :keyword:`!else` clause, if present, is
+executed, and the loop terminates.
```

Comment:
This change mostly is in formatting.  Can you please make it *minimal*, probaly just one string?

> The expression following ``in`` is evaluated once

Maybe it's better to reference starred_expression_list?